django_etebase/exceptions.py

```diff
@@ -7,3 +10,39 @@ def __init__(self, code, detail, status_code=status.HTTP_400_BAD_REQUEST):
             {"code": code, "detail": detail,}
         )
         self.status_code = status_code
+
+
+class AuthenticationFailed(HTTPException):
```

Comment:
All of the permissions are missing the `code` for the actual textual code of how we call the error (matching what's there in `detail`).

Reply:
ill try to have fastAPI return this when we raise custom exceptions  
{"code": "bad_user", "detail": " The user is bad"}

Reply:
Not urgent to address this at the moment. Just wanted to share my thoughts since I already took a look.
The status_code should be used for the actual HTTP status code of the reply.

Reply:
I think this is fixed now, right? If so, please resolve the thread.